solutions/python/02-cz2/diff/app/main.py.diff

```diff
@@ -0,0 +1,16 @@
+@@ -1,13 +1,14 @@
+ import sys
+
+
+ def main():
+     sys.stdout.write("$ ")
+     sys.stdout.flush()
+
+     # Wait for user input
+-    input()
++    command = input()
++    print(f"{command}: command not found", flush=True)
```

Comment:
Hmm, not sure if we actually need `flush=True` here. Our test runners use "PYTHONUNBUFFERED=true" so certainly don't need it. @andy1li can we try adding the `-u` flag to the starter instead? That way we can avoid the `flush()` calls above and here.